### PR TITLE
Scons fix

### PR DESCRIPTION
**Pull request by BillyLiggins**
That fix.

### CODE REVIEW

File: src/SConscript [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!python
 import os
+from SCons.Script import Clean
```
Comment (jackdunger) [inline src/SConscript]: do you also need to `import Glob` or did this work without?

**Comment by jackdunger** [comment]
Thanks! Bit confused as to why there's three commits?

**Comment by BillyLiggins** [comment]
3 commits because I added the updated simpleFit from #54  and then removed it.

**Comment by jackdunger** [comment]
Ok gotcha, merging

**Comment by BillyLiggins** [comment]
:+1:

**Pull request merged**